### PR TITLE
Add "safe" filter à la Nunjucks

### PR DESCRIPTION
**Pull request by dz4k**
Per Discord discussion:

> dz4k — Yesterday at 10:51 PM \
> @Óscar there should be a filter like `trustHtml` that simply marks text as trusted without transforming it, and the docs should be updated to recommend that when autoescaping is used. Happy to PR
>
> Óscar — Yesterday at 11:00 PM \
> I see.
>
> Ok, maybe `safe`, like Nunjucks.

Note: I never liked the `safe` name but it might be better to keep it for familiarity.

### CODE REVIEW

File: src/environment.ts [hunk 1]
```diff
@@ -232,7 +232,7 @@ export class Environment {
           })`;
         }
       } else {
-        if (name === "unescape") {
+        if (name === "unescape" || name === "safe") {
```
Comment (oscarotero) [inline src/environment.ts:235]: I think only `safe` filter should be taken into account here and treat `unescape` as any other filter, without any special behavior.

File: src/environment.ts [hunk 2]
```diff
@@ -232,7 +232,7 @@ export class Environment {
...
           unescaped = true;
```
Comment (oscarotero) [inline src/environment.ts:236]: If we include a `continue` here, we no need to create a plugin with an empty function. `safe` can be considered a core feature of Vento, in the same way as `autoescape` option.

**Review by oscarotero — CHANGES_REQUESTED** [review]
I'm open to pick a different name, but I wouldn't mention `html` here (like your proposal `trustHtml`) because you may want to escape non-HTML code.

Maybe `no_autoescape` is more clear in its purpose, but it's also more longer.

**Comment by oscarotero** [comment]
Hi @dz4k 
Looks like there're some conflicts due the latest changes in the documentation.
Can you fix them? I also left a couple of suggestions to simplify your changes. Let me know if you have any question.
Thanks!

**Comment by dz4k** [comment]
@oscarotero just got around to making those changes. Any final verdict on the filter name? maybe `noescape` or `trust`, or stick with `safe`?

BTW: 

> because you may want to escape non-HTML code.

It would be cool if the `autoescape` filter accepted an escaping function, so we could use Vento for templating other kinds of files without causing syntax errors -- maybe a future feature

**Comment by oscarotero** [comment]
> Any final verdict on the filter name? maybe noescape or trust, or stick with safe?

`noescape` is confusing because you can combine it with `escape` filter: `{{ string | escape | noescape }}`.

If you don't like `safe`, I'm okay with `trust` (it's also short).
Let me know if you want to change it and when can I merge this.

> It would be cool if the autoescape filter accepted an escaping function, 

Maybe this function can be configurable in the options:

```js
const env = vento({
    autoescape: true, // use the default function
});

const env = vento({
    autoescape: (string) => string, // use a custom function
});
```

**Comment by dz4k** [comment]
`safe` is fine, feel free to merge

**Comment by oscarotero** [comment]
Great. Thanks!

**Pull request merged**